features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/SessionEndedMetric.kt

```diff
@@ -37,6 +39,8 @@ internal class SessionEndedMetric(
     private var lastTrackedView: TrackedView? = null
     private var wasStopped: Boolean = false
 
+    private var uploadQuality: MutableMap<String, MutableMap<UploadQualityMetric, Any?>> = mutableMapOf()
```

Comment:
seems like it can be `val`?

```suggestion
    private val uploadQuality = mutableMapOf<String, MutableMap<UploadQualityMetric, Any?>>()
```